pyproject.toml

```diff
@@ -74,6 +74,9 @@ python-telegram-bot = { version = "~=21.3", extras = ["all"], optional = true }
 opentelemetry-instrumentation = { version = "*", optional = true }
 sqlalchemy = { version = "*", extras = ["asyncio"], optional = true }
 opentelemetry-exporter-otlp = { version = ">=1.20.0", optional = true }  # log body serialization is required
+pyyaml = "^6.0.1"
+fastapi = "^0.112.1"
+uvicorn = "^0.30.6"
```

Comment:
Dependency versions are only locked if there's a reason (otlp only added log body serialization after 1.20; telegram uses specific features only available in the latest release).
Otherwise, version should not be locked.